src/bindgen/utilities.rs

```diff
@@ -83,9 +83,9 @@ macro_rules! syn_item_match_helper {
             syn::Item::Type(ref item) => (|$i: &syn::ItemType| $a)(item),
             syn::Item::Union(ref item) => (|$i: &syn::ItemUnion| $a)(item),
             syn::Item::Use(ref item) => (|$i: &syn::ItemUse| $a)(item),
-            syn::Item::Existential(ref item) => (|$i: &syn::ItemExistential| $a)(item),
             syn::Item::TraitAlias(ref item) => (|$i: &syn::ItemTraitAlias| $a)(item),
             syn::Item::Verbatim(_) => (|| $b)(),
+            _ => panic!("Unhandled syn::Item:  {:?}", $s),
```

Comment:
I'd prefer to keep handling the others.

Reply:
Unfortunately https://docs.rs/syn/1.0.3/src/syn/item.rs.html#79